src/pages/board.tsx

```diff
@@ -0,0 +1,22 @@
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '@/src/styles/Home.module.css'
+import Header from './components/Header'
+import BestPostList from './components/BestPostList'
+import PostList from './components/PostList'
+
+export default function board() {
+  return (
+    <>
+      <Head>
+        <title>판다마켓</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/assets/icons/favicon.ico" />
+      </Head>
+      <Header></Header>
+      <BestPostList></BestPostList>
+      <PostList></PostList>
```

Comment:
팀마다 컨벤션과 개인 성향은 다르겠지만 개인적으로는 children이 없다면 
```suggestion
      <Header />
      <BestPostList />
      <PostList />
```
으로 해주는 것을 선호하는 편입니다!